hstream-store/admin/HStream/Store/Admin/Command/Logs.hs

```diff
@@ -0,0 +1,53 @@
+module HStream.Store.Admin.Command.Logs
+  ( runLogsCmd
+  ) where
+-- module HStream.Store.Admin.Command.Logs where
+import           Data.Bits                        (shiftR, (.&.))
+import           System.IO.Unsafe                 (unsafePerformIO)
+import           Z.IO.Time                        (SystemTime (..),
+                                                   formatSystemTime,
+                                                   simpleDateFormat)
+
+import qualified HStream.Store                    as S
+import           HStream.Store.Admin.Types
+import qualified HStream.Store.Internal.LogDevice as S
+
+client :: S.LDClient
+client = unsafePerformIO $ do
+  _ <- S.setLogDeviceDbgLevel S.C_DBG_ERROR
+  S.newLDClient "/data/store/logdevice.conf"
+{-# NOINLINE client #-}
+
+runLogsCmd :: LogsSubCmd -> IO ()
+runLogsCmd (InfoCmd logid) = runLogsInfo logid
+runLogsCmd ShowCmd         = putStrLn "log show"
```

Comment:
`runLogsCmd ShowCmd` should be `undefined` or an error to indicate that this is "Not Implemented"